builtin/logical/aws/backend.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/service/iam/iamiface"
 	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+
```

Comment:
```suggestion
```

Reply:
Not sure if this space is intentional?

Reply:
It got added by Goland when sorting imports. I removed it in the latest commit